resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesClusterSchedulerBackend.scala

```diff
@@ -191,6 +198,7 @@ private[spark] class KubernetesClusterSchedulerBackend(
           }
         }
       }
+      updateJobResourceStatus(STATUS_CURRENT_EXECUTORS, totalRegisteredExecutors.get())
```

Comment:
should this be flagged off with a feature flag in the first merge?  I want to make sure we're not inadvertently introducing a dependency on a newer version of k8s than what we aim to currently support (both 1.5 and 1.6)